x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/SqlProtocolTestCase.java

```diff
@@ -67,11 +67,16 @@ public void testTextualType() throws IOException {
     
     public void testDateTimes() throws IOException {
         assertQuery("SELECT CAST('2019-01-14T12:29:25.000Z' AS DATETIME)", "CAST('2019-01-14T12:29:25.000Z' AS DATETIME)",
-            "datetime", "2019-01-14T12:29:25.000Z", 24);
+            "datetime", "2019-01-14T12:29:25.000Z", 35);
```

Comment:
Since the precision changed, please change the tests' values or add new tests (preferred) to reflect the increased precision.

Reply:
Unfortunately this revealed another bug: https://github.com/elastic/elasticsearch/issues/40692
so currently I cannot have easily a test with custom timezone.